ext/Cargo.toml

```diff
@@ -16,20 +16,31 @@ ruby-api = []
 
 [dependencies]
 lazy_static = "1.4.0"
-magnus = { version = "0.5.3", features = ["rb-sys-interop"] }
-rb-sys = "~0.9.81"
-wasmtime = { version = "= 9.0.1" }
-wasmtime-wasi = "= 9.0.1"
-wasi-common = "= 9.0.1"
-wasi-cap-std-sync = "= 9.0.1"
+# For ruby-head support, we have to lock to 0.5.4 until
+# rb-sys/stable-api-compiled-fallback feature is used in Magnus.
+magnus = { version = "= 0.5.5", features = ["rb-sys-interop"] }
+rb-sys = { version = "*", default-features = false, features = [
+  "stable-api-compiled-fallback",
+] }
+wasmtime = { version = "= 9.0.4" }
+wasmtime-wasi = "= 9.0.4"
+wasi-common = "= 9.0.4"
+wasi-cap-std-sync = "= 9.0.4"
 cap-std = "1.0.5"
 anyhow = "*" # Use whatever Wasmtime uses
-wat = "1.0.69"
-tokio = { version = "1.28.2", features = ["rt", "rt-multi-thread", "time", "net"], optional = true }
-async-timer = { version = "1.0.0-beta.8", features = ["tokio1"], optional = true }
+wat = "1.0.63"
```

Comment:
Any reason why we went back to `1.0.63`?

Reply:
Git conflict resolution gone wrong, fixed now